src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/memory.html.twig

```diff
@@ -16,9 +16,9 @@
 
         <div class="sf-toolbar-info-piece">
             <b>PHP memory limit</b>
-            <span>{{ collector.memoryLimit == -1 ? '&infin;' : '%.0f'|format(collector.memoryLimit / 1024 / 1024)|escape }} MB</span>
+            <span>{{ collector.memoryLimit == -1 ? 'Unlimited' : '%.0f MB'|format(collector.memoryLimit / 1024 / 1024)|escape }}</span>
```

Comment:
you can remove the explicit escaping now, given that the raw string on the left does not contain HTML entities anymore and so the autoescaping would not break the output anymore when escaping the whole ternary statement